src/test/java/org/opensearch/performanceanalyzer/reader/ReaderMetricsProcessorTests.java

```diff
@@ -79,6 +81,7 @@ public void testShardBulkParseAndQuery() throws Exception {
                 assertNotNull(record.get("ShardBulkDocs"));
                 assertEquals(1519.0, (Double) record.get("ShardEvents"), 0.0);
                 assertEquals(507096.0, (Double) record.get("ShardBulkDocs"), 0.0);
+                assertTrue((Double) record.get("CPU_Utilization") > 1.0);
```

Comment:
Let us check for the exact value instead of generic ` > 1.0`

Reply:
@khushbr 
It might be tricky to get an absolute value if the time of the event when the CPU spikes and taking of the measurements don't align. But yes, it can't be a simple check as just `> 1.0` and should probably test for a range (or a lower bound range).

Reply:
Yes you're both right, my train of thought was that the 1.0 is relatively high for this metric and this type of workload and that asserting for `>1.0` looks good as one of the test files is full of shardbulk events. I'll find a fitting range for it.

Reply:
@khushbr @kkhatua I've come up with a delta value, empirically, please raise any concerns if it seems too big/small.